pom.client.xml

```diff
@@ -748,8 +747,7 @@
     <module>./sdk/eventhubs/azure-eventhubs</module>
     <module>./sdk/keyvault/azure-keyvault-secrets</module>
     <module>./sdk/keyvault/azure-keyvault-keys</module>
-    <!-- As recommended by Eng Sys team, removing tracing as it is built and released separately -->
-    <!--    <module>./sdk/tracing</module>-->
+<!--    <module>./sdk/tracing/tracing-opentelemetry</module>-->
```

Comment:
Did you mean to un-comment this line?

Reply:
yeah don't think this was related to the javadoc issue we were trying to fix. Want to leave this commented for now.